### PR TITLE
Adding build script, dockerfile, json for Odoo 16.0

### PR DESCRIPTION
**Pull request by vishakadesai**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: o/odoo/odoo_16.0_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+# Install dependencies
+python3.10 -m pip install pip wheel
+python3.10 -m pip install -r requirements.txt
+
+# Start Odoo server, refer README
+# python3.10 odoo-bin -d mydb -r odoo -w odoo --db_host 172.21.0.2 --db_port 5432 -i INIT
```
Comment (kandarpamalipeddi) [inline o/odoo/odoo_16.0_ubi_8.5.sh:57]: @vishakadesai 
How about running test suite.
Please find the latest test suite log at : http://runbot143.odoo.com/runbot/static/build/22731962-16-0/logs/run.txt

They are using below command to execute the test-suite:
`python3 odoo/odoo-bin --addons-path odoo/addons,odoo/odoo/addons --workers 2 --gevent-port 8070 --max-cron-threads 1 -d <dir-name> --proxy-mode --db-filter %d.*$ --smtp 172.17.0.1`

Can you please check it once ?
CC: @seth-priya, @nikhil-kalbande

File: o/odoo/build_info.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "odoo",
+    "github_url":"https://github.com/odoo/odoo",
+    "version": "16.0",
+    "package_dir": "o/odoo/",
+
+    "16.0" : {
+        "dir":"16.0_debian_bullseye",
+        "build_script" :"odoo_16.0_ubi_8.5.sh"
```
Comment (kandarpamalipeddi) [inline o/odoo/build_info.json:9]: @vishakadesai 
As you are using `debian:bullseye-slim` as base image, please provide the `base_docker_image` tag as `debian:bullseye-slim` and `base_docker_variant` as `ubuntu`.
**We need to provide these 2 details for all the non UBI8 based docker images.**

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

**Comment by vishakadesai** [comment]
@kandarpamalipeddi I have resolved the comments you raised
Added the additional tags to the build_info.json file and the command to run the test suite in the build script
Have also made necessary additions to README
Requesting you to review when possible

CC: @nikhil-kalbande @seth-priya

**Comment by kandarpamalipeddi** [comment]
@vishakadesai 
As we are not supporting Docker inside travis check, we can go ahead with this PR.
Once we support docker inside travis check, we need to revisit this PR and enable testing.
@seth-priya, @nikhil-kalbande

**Review by kandarpamalipeddi — APPROVED** [review]

**Pull request merged**